docs/getting-started-guides/docker.md

```diff
@@ -73,7 +73,7 @@ variable.
 For example, OS X:
 
 ```shell
-wget http://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/darwin/amd64/kubectl
+curl -O http://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/darwin/amd64/kubectl
```

Comment:
Oh, thanks! Sorry, didn't get that from the commit title and thought it wasn't on purpose :)